hearpreprocess/pipeline.py

```diff
@@ -38,16 +40,6 @@
 TRAIN_PERCENTAGE = 100 - VALIDATION_PERCENTAGE - TEST_PERCENTAGE
 TRAINVAL_PERCENTAGE = TRAIN_PERCENTAGE + VALIDATION_PERCENTAGE
 
-# We want no more than 5 hours of audio (training + validation) per task.
-# This can be overriden in the task config.
-# e.g. speech_commands test set.
-# If None, no limit is used.
-MAX_TASK_DURATION_BY_SPLIT = {
-    "train": 3600 * 5 * TRAIN_PERCENTAGE / TRAINVAL_PERCENTAGE,
-    "valid": 3600 * 5 * VALIDATION_PERCENTAGE / TRAINVAL_PERCENTAGE,
-    "test": 3600 * 5 * TEST_PERCENTAGE / TRAINVAL_PERCENTAGE,
-}
```

Comment:
Getting rid of this - we don't use it, and better to specifically set the duration for each split